### PR TITLE
Added more cases to handlePodcastMessages for Android/iOS Podcast Player

### PR DESCRIPTION
**Pull request by rt4914**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [x] Optimization
- [ ] Documentation Update

## Related Tickets & Documents

- Related Issue #
- Closes #

## How to test this PR?

1. Checkout the code of this PR https://github.com/forem/ForemWebView-android/pull/56 locally.
2. Checkout in this current branch `rt4914/add-forem-mobile-podcast-options` (basically this PR) locally.
3. Using ngrok fetch the https URL for the local environment
4. Update the url in `env` file.
5. Now set this URL in android code https://github.com/forem/ForemWebView-android/blob/e29fd4f61fbdc8eeb9699a3342f29f9b13e95f4e/app/src/main/java/com/foremlibrary/app/MainActivity.kt#L35
6. Run the android app and play any podcast.
7. From podcast notification, pause the podcast and notice that this change is reflected in Android app too.

(For simplicity you can check the below output.)

## QA Instructions, Screenshots, Recordings

https://user-images.githubusercontent.com/9396084/184368290-38ba78db-f11f-43e5-b2c4-5bc13a52c7d1.mp4

### UI accessibility concerns?
No

## Added/updated tests?

- [ ] Yes
- [x] No
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

**Review by fdocr — COMMENTED** [review]
I think this looks great!

You're right that we should implement this on iOS as well. At the moment we're only [pausing the player](https://github.com/forem/ForemWebView-ios/blob/main/Sources/ForemWebView/ForemMediaManager/ForemMediaManager%2BLockedScreenControls.swift#L20) there and it should reflect the icon when you return to the app.

It shouldn't break the iOS app IMO, so this could be shipped without having to wait for a joint release of both apps.

I wasn't able to fully test this on a device (it's been a while and I ran into some issues with the phone I was using). I'll approve after I'm able to sort this out early next week.

**Review by fdocr — APPROVED** [review]
I was now able to test this on an actual device and works great 👍🏼

**Review by joshpuetz — APPROVED** [review]

**Pull request merged**
